content/en/docs/Tasks/adding-an-admission-webhook.md

```diff
@@ -0,0 +1,83 @@
+# Admission Webhooks
+
+## Background
+
+OLM is capable of managing the lifecycle of [validating](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#validatingadmissionwebhook) and [mutating](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#mutatingadmissionwebhook) admission webhooks that are shipped alongside your operator.
+
+You may be unfamiliar with webhooks. At a high level, after a request has been authenticated and authorized, admission webhooks intercept requests against the Kubernetes API and have an opportunity to validate or update the object before it is saved in the object store. Please refer to the following table that highlights what each webhook is capable of:
+
+|                    | Validating Webhook | Mutating Webhook |
+|--------------------|--------------------|------------------|
+| Validating Objects |          x         |         x        |
+| Mutating Objects   |                    |         x        |
+
+If you are interested in learning more about admission webhooks, please review the [official kubernetes documentation](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#what-are-they).
+
+## Defining your Webhook in the ClusterServiceVersion
+
+### Creating a Webhook
+
+It is outside the scope of this document to define the process of creating an admission webhook, however there are a few constraints that you should be aware of before developing your webhook if you plan to have it managed by OLM:
```

Comment:
Again, instead of saying "it is outside of the scope of this document`, we could direct readers to the References section, that can have links to resources readers can refer to for creating admission webhook.